XamlControlsGallery/ControlPagesSampleCode/NumberBox/NumberBoxSample3_xaml.txt

```diff
@@ -0,0 +1 @@
+﻿<muxc:NumberBox x:Name="FormattedNumberBox" Header="Enter an dollar amount:" PlaceholderText="0.00" />
```

Comment:
As above, please move this sample inline.

Reply:
Since this sample also has a CS txt file, I can see an argument to keep the xaml in a txt file as well.